### DISCUSSION TITLE
How to manipulate each selection as in normal mode?

### DISCUSSION THREAD

**Discussion by Andrew15-5** [smoka7/multicursors.nvim#51] [Q&A]
Currently, AFAIU, I can only write something on the left side or on the right side of the selections. I can also execute a macro on each selection (but I can only pass the macro register instead of the macro itself). I can also cut/delete/yank selections. But I can't uppercase selections (as in `U`) which is what I'm trying to do.

Make

```vim
<cr>
<cr>
<cr>
<cr>
```

like

```vim
<CR>
<CR>
<CR>
<CR>
```

**Comment by smoka7** [comment]
Uppercase and lowercase isn't implemented yet.
